tools/pkglint/lib/rules.ts

```diff
@@ -46,9 +46,32 @@ export class DescriptionIsRequired extends ValidationRule {
   }
 }
 
+/**
+ * Verify that all packages have a publishConfig with a publish tag set.
+ */
+export class PublishConfigTagIsRequired extends ValidationRule {
```

Comment:
nit: I would've just called it `PublishConfig` or `NpmDistTag` but I see you've picked up the naming from the surrounding classes. Take a look at some others later in the file too.